src/com/facebook/buck/android/AndroidLibraryDescription.java

```diff
@@ -153,6 +154,8 @@ public void findDepsForTargetFromConstructorArgs(
     Optional<Query> getDepsQuery();
 
     Optional<Query> getProvidedDepsQuery();
+
+    ImmutableList<String> getExtraKotlincArguments();
```

Comment:
Kotlin library already has `extra_arguments` which pass arguments to the kotlin compiler. Android library is different since it invokes two compilers.